From 578b57b0f40da9d4aa07f21445aa9e19d00a4f77 Mon Sep 17 00:00:00 2001
From: Ralf Gommers <ralf.gommers@gmail.com>
Date: Sun, 9 Apr 2023 13:00:51 +0100
Subject: [PATCH 6/9] BUG: include `npy_common.h` in `npy_cblas.h`

This is necessary, because it uses `npy_int64` & co. The implicit
assumption previously was that this was made available in the
file that included `npy_cblas.h`, but this wasn't guaranteed in
any way. This is why the build failed after removing a
`from numpy.math cimport` line in `_ellip_harm.pxd`.
---
 scipy/_build_utils/src/npy_cblas.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scipy/_build_utils/src/npy_cblas.h b/scipy/_build_utils/src/npy_cblas.h
index 751854b6e..02961d430 100644
--- a/scipy/_build_utils/src/npy_cblas.h
+++ b/scipy/_build_utils/src/npy_cblas.h
@@ -6,6 +6,7 @@
 #ifndef NUMPY_CORE_SRC_COMMON_NPY_CBLAS_H_
 #define NUMPY_CORE_SRC_COMMON_NPY_CBLAS_H_
 
+#include <numpy/npy_common.h>
 #include <stddef.h>
 
 /* Allow the use in C++ code.  */
